src/waypoint.jsx

```diff
@@ -169,8 +167,8 @@ export default class Waypoint extends React.Component {
     if (node.offsetParent === this.scrollableAncestor || !node.offsetParent) {
       return node.offsetTop;
     } else {
-      return node.offsetTop +
-        this._distanceToTopOfScrollableAncestor(node.offsetParent);
+      let nextOffset = this._distanceToTopOfScrollableAncestor(node.offsetParent);
```

Comment:
Weird that that wasn't picked up... ah, when you run the test script the eslint is run once initially and then karma runs continuously - I hacked away until the tests were green and I was happy with the code, but never ran the tests from scratch after that.